lib/Helper/Package.ts

```diff
@@ -59,3 +59,20 @@ function fulfillsVersionRange(
       : satisfies(cleanedUserVersion, acceptableVersions))
   );
 }
+
+/**
+ * Determines if the passed `package.json` object has the passed package installed.
+ *
+ * @param appPackage The `package.json` object
+ * @param packageName The name of the package to check for
+ *
+ * @returns `true` if the package is installed, `false` otherwise
+ */
+export function hasPackageInstalled(
```

Comment:
Extracted this from the NextJS integration class. Will probably extract more methods going forward as the initial setup is very similar